.gitignore

```diff
@@ -4,3 +4,4 @@
 /tests/Jane/generated/*
 !/tests/Jane/generated/.gitkeep
 .php_cs.cache
```

Comment:
This should not be needed anymore, as it's the name of the cache file from PHP-CS-Fixer 2 (see: [upgrade docs](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/3.0/UPGRADE-v3.md#rename-of-files))

Reply:
Would you mind sending a PR? :)

Reply:
Here it is: #85